daemon/logger/factory.go

```diff
@@ -56,10 +64,12 @@ func (lf *logdriverFactory) get(name string) (Creator, error) {
 	defer lf.m.Unlock()
 
 	c, ok := lf.registry[name]
-	if !ok {
-		return c, fmt.Errorf("logger: no log driver named '%s' is registered", name)
+	if ok {
+		return c, nil
 	}
-	return c, nil
+
+	c, err := getPlugin(name, plugingetter.Acquire)
```

Comment:
Looks like the one in `Close` https://github.com/docker/docker/pull/28403/files#diff-9d020faaa9beb9f01b9b34ca5cb795bbR72